moirae/models/base.py

```diff
@@ -427,20 +427,6 @@ class OutputQuantities(GeneralContainer):
         Field(description='Voltage output of a battery cell/model. Units: V')
 
 
-class TransientVector(GeneralContainer):
-    """
-    Stores physical transient/instantenous hidden state
-    """
-    pass
-
-
-class AdvancedStateOfHealth(HealthVariable):
-    """
-    Stores A-SOH
-    """
-    pass
-
-
```

Comment:
I think removing the `TransientVector` is good. Now, I think we should have a discussion on the A-SOH part. While the A-SOH is, as the name implies, a `HealthVariable` on its own, containing other `HealthVariables`, it is one additional thing on top of that: it is the **necessary and sufficient** set of `HealthVariable`s to fully reproduce an IVT(t) profile. So, I believe there are merits to keeping these classes distinct, even if there is no added functionality in the A-SOH. I can be convinced otherwise though, so let's talk about this when we meet later today.